docs/pages/en-us/Client-API.md

```diff
@@ -108,28 +109,31 @@ In the [`ConfigurationSource`](#ConfigurationSource) above, we used JSON-encoded
 - `CAF_CONFIG_PATH: /etc/sampleapp/config`
 - `CAF_APPNAME: sampleappgroup/sampleapp`
 
-Given this configuration, you would configure Auditing by creating a file named `cfg_sampleappgroup_sampleapp_KafkaAuditConfiguration` in the `/etc/sampleapp/config/` directory. The contents of this file should be similar to the following:
+Given this configuration, you would configure Auditing by creating a file named `cfg_sampleappgroup_sampleapp_ElasticsearchAuditConfiguration` in the `/etc/sampleapp/config/` directory. The contents of this file should be similar to the following:
```

Comment:
Name of configuration file should be based on class name so it should be ElasticAuditConfiguration and not ElasticsearcAuditConfiguration.